### ISSUE TITLE
TeamsDemo.Office.com

### ISSUE THREAD

**Issue by Shuwi75** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#748]
Another great resource that seems to be lacking from this list: http://teamsdemo.office.com/

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5eafa2de-df3c-6c8e-50cf-71ef0fc5cfb0
* Version Independent ID: d47ba848-f74d-29d2-cd22-de76af06c410
* Content: [End user training for Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/enduser-training#feedback)
* Content Source: [Teams/enduser-training.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/enduser-training.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by AndreaBarr** [comment]
@Shuwi75 Thank you for this feedback

I am sending this information to the Teams team so we can get this added. Thank you again for taking the time to help us make the documents better. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
